starter-packs/refinery-deployment/cloudformation-aws/image-builder/generate_mappings.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
```

Comment:
```suggestion
set -exo pipefail

```

Reply:
Thanks!  I pulled this in, but without the -x because the debug output made the output unusable.